.github/assets/test-manifest.yml

```diff
@@ -0,0 +1,111 @@
+# Namespace
```

Comment:
I'd change the name of this file to be more descriptive, for example `test-upgrade-manifest.yaml` or something similar.